tests/utils/meilisearch-test-utils.ts

```diff
@@ -51,7 +51,7 @@ async function getClient(permission: string): Promise<MeiliSearch> {
     return anonymousClient
   }
 
-  if (permission === 'Public') {
+  if (permission === 'Search') {
     const searchKey = await getKey(permission)
     const publicClient = new MeiliSearch({
```

Comment:
Can you change this to `searchClient` ? It also needs to be changed line 60

Reply:
I have made this change